### PR TITLE
[MRG] Adds background

### PR DESCRIPTION
**Pull request by christophM**

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -2600,7 +2600,7 @@
 /* One */
 
 	#one:before {
-		background-image: url("../../images/banner.jpg");
+		background-image: url("https://www.dropbox.com/s/dax4izyz0x0h5mt/space.jpg?dl=1");
```
Comment (betatim) [inline assets/css/main.css]: should we use dropbox or in the long term move assets here?

File: index.html [hunk 2]
```diff
@@ -18,9 +18,9 @@
 		<!-- Header -->
 			<section id="header">
 				<header>
-					<span class="image avatar"><img src="images/avatar.jpg" alt="" /></span>
+					<!-- <span class="image avatar"><img src="images/avatar.jpg" alt="" /></span> -->
```
Comment (betatim) [inline index.html]: can we not delete it instead f commenting out? If we need to go back we have `git` :)

Comment (christophM) [reply index.html]: done

File: assets/css/main.css (hunk 1 above)
Comment (christophM) [reply assets/css/main.css]: images moved to repo

**Pull request merged**